data/en/sections/custom.yml

```diff
@@ -0,0 +1,12 @@
+# section information
```

Comment:
@hossainemruz this creates an actual section on the homepage
![image](https://user-images.githubusercontent.com/8029578/97767015-cb955700-1aef-11eb-8248-8f56b012b25e.png)

I think it would be good for the site to have a real example for people to refer to and see, but i'd understand if you don't want this on your homepage. If you have any thoughts on what a better example would be I'd be happy to implement that (maybe something with some nice looking cards or something)

Reply:
Oh, looking at the original issue https://github.com/hossainemruz/toha/issues/87, maybe an image gallery would be good. I could pull something from https://mdbootstrap.com/plugins/jquery/gallery/. I'm not the best at the design/visual aspect so if you have any recommendations I'd be happy to take them

Reply:
Since, this repo is for documentation purpose, many examples we can not show here to keep this site clean. So, I am planning to create a separate repo that will serve as standard example. We will show all the features with minimal configuration. People, can just fork that repo.

I will probably create a separate organization for Toha theme. Then, rename this repo to `guides` or `docs` and move into that organization.That organization will contains the following repos:

- `hugo-toha/toha` (current `hossainemruz/toha`)
- `hugo-toha/guides` (current `hossainemruz/toha-example-site`)
- `hugo-toha/hugo-toha.github.io` (example site which will also server as example of uisng this theme in Github pages)

For now, you can add a [Contact Me](https://bootsnipp.com/snippets/83Br) form for custom section example and `about us` page for custom page example. Don't worry about the visual I will work on it. Also, you don't have to make `Contact Me` section functional. Just visual part should be enough for now.

Feel free to add whatever example you want. I am just suggesting.

Reply:
Sounds good, I updated to use the contact me for now
![image](https://user-images.githubusercontent.com/8029578/97779910-a2101600-1b57-11eb-8523-4f41699d4909.png)

Reply:
Hey Alex, there is no layouts folder in https://github.com/hugo-toha/hugo-toha.github.io  repository, so I made one and also the sub-folders.
But where to include the css files ?

Reply:
Hey @iampankajsingh everything for this form is in this PR. There wasn't any special css since I just used the bootstrap classes already pulled into the site.

if you are looking to add custom css, I believe the convention is to provide your own style.css file like this denotes https://github.com/hugo-toha/toha/blob/1d0985d14c/static/assets/css/style.css#L1 thats what i did with my site atleast https://github.com/alex-bezek/blog/blob/master/static/assets/css/style.css

Reply:
yeah got it to work by keeping in static/asset/...
but still deployed code in main branch doesn't have static folder, and I get this error in console.
GET https://iampankajsingh.github.io/static/assets/css/sections/skills.css 404 not found.

@hossainemruz , if we use custom section, do we see those folder changes in main branch ? coz I can't see my static and layouts folder which I created in source branch.
It works fine though, apart from that error.

Reply:
Currently, you can only overwrite `style.css`. It has been designed to overwrite by the users. ref: https://github.com/hugo-toha/toha/blob/5e1e78c846277df232751a81e48874599f0a63fc/static/assets/css/style.css#L1

You can't put an entire html page for a custom section like this: https://github.com/iampankajsingh/iampankajsingh.github.io/blob/03e1de32984d13d2a1c1be32a5a41a17010abebd/layouts/partials/sections/custom.html#L1

Reply:
Btw, give us some time. Wait until we merge this PR.

Reply:
> Btw, give us some time. Wait until we merge this PR.

I'm ok if @ iampankajsingh keeps asking questions, it's good early feedback about how thorough my docs are (clearly as it is right now it doesn't answer all the questions). I'll update this pr today with
* Notes about styling by overriding the style.css (NOT by adding a new custom css file)
* explicitly note that if the subfolders don't exist yet (which they wouldn't unless you have done one of this customizations already) then you should add them
* perhaps provide a recap/overview of the final file structure denoting new and modified files